vendor/vendor.json

```diff
@@ -137,12 +137,6 @@
 			"revision": "504f88b65b8e68b7c6cd0507c9788b6518d7cc1b",
 			"revisionTime": "2019-04-26T09:22:21Z"
 		},
-		{
-			"checksumSHA1": "aR8giSwNg7ogfWoUuEqkqmN/HRc=",
-			"path": "github.com/b4b4r07/go-pipe",
-			"revision": "ed1c7948fefc784c9e562b891ee421bb0d803811",
-			"revisionTime": "2019-05-10T19:03:01Z"
-		},
```

Comment:
So that's a left-over from the changes in cc5fb9b, isn't it? You could reference that commit in commit message.

Reply:
Commit messages have been updated in a3622f0 and name added to NOTICE file in a70c4b90255a43692e404ce1ce3a481136430ee6.